nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -23,51 +23,102 @@ in
         defaultText = lib.literalExpression "pkgs.wireplumber";
         description = lib.mdDoc "The wireplumber derivation to use.";
       };
-    };
-  };
 
-  config = lib.mkIf cfg.enable {
-    assertions = [
-      {
-        assertion = !config.hardware.bluetooth.hsphfpd.enable;
-        message = "Using Wireplumber conflicts with hsphfpd, as it provides the same functionality. `hardware.bluetooth.hsphfpd.enable` needs be set to false";
-      }
-    ];
+      configPackages = lib.mkOption {
+        type = lib.types.listOf lib.types.package;
+        default = [ ];
+        description = lib.mdDoc ''
+          List of packages that provide wireplumber configuration, in the form of
```

Comment:
Nit: WirePlumber (capitalization is already all over the place but still no point making it worse)

Reply:
Do you think having an additional commit to fix the pre-existing capitalisation typos would be good? Otherwise I'll just fix the ones I made and rebase. Sorry by the way, for the descriptions having turned out a bit messy

Reply:
If you want to do it, go ahead